apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/entity/GrayReleaseRule.java

```diff
@@ -26,9 +26,9 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name = "GrayReleaseRule")
-@SQLDelete(sql = "Update GrayReleaseRule set IsDeleted = 1, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000) where Id = ?")
-@Where(clause = "isDeleted = 0")
+@Table(name = "`GrayReleaseRule`")
+@SQLDelete(sql = "Update \"GrayReleaseRule\" set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000) where Id = ?")
```

Comment:
@nobodyiam Yes, otherwise the sql will use GRAYRELEASERULE table. And we can't use **`** in sql

Reply:
I think one basic rule is to keep the code consistent so that other contributors could know what to do in the future.
For table names like 'Release', 'Namespace' that conflict with database keywords, I suppose we need to quote them to avoid syntax error?
But for table names like 'GrayReleaseRule', I'm not sure whether we need to quote them to avoid case sensitivity issue. If it's necessary, then we also need to quote other table names like 'App', 'Commit', etc.